### ISSUE TITLE
Google Analytics not importing all data

### ISSUE THREAD

**Issue by tech4him1** [Fyrd/caniuse#3339]
When I import my data from one day into caniuse, it shows:

<pre>
<div class="ga-import-details"><h4>Import details</h4><p>Total imported: 38.89%</p><p>Total untracked: 61.11%</p><table class="ga-tracked-table"><caption>Browsers included</caption><tr class="ga-tracked-item"><td>Chrome 55</td><td>13.89%</td></tr></table><table class="ga-untracked-table"><caption>Browsers not included</caption><tr class="ga-untracked-item"><td>Mozilla Compatible Agent 5.0</td><td>61.11%</td></tr></table></div>
</pre>

However, when I look in my Analytics console or use the API, it shows:

  Browser       |             Browser Version  | Sessions
  -------------------------- | ----------------- | ----------
  Android Browser     |       4.0          |     1
  Chrome                    | 55.0.2883.87 |      3
  Chrome                     |55.0.2883.91  |    1
  Firefox                    |51.0       |       1
  Internet Explorer     |     11.0  |            3
  Mozilla Compatible Agent|   5.0 |              8
  Safari                 |    10.0       |       1
  Safari                  |   10.0.2     |       1
  Safari                   |  8.0          |     1
  Safari                    | 9.0           |    1

**Comment by Fyrd** [comment]
Hmm, that's very strange. Can't reproduce that with my data. Do the time frames in both cases match? Have you tried it with different time frames?

**Comment by tech4him1** [comment]
Yes, I have.  For example, no matter how big of a timeframe I pick around it, I never get any "Android Browser" instances imported.

I couldn't find the code that you use to get that data, is it open-source?  If I can have a copy, I can test it and see if I find any bugs.
